tests/components/justnimbus/test_config_flow.py

```diff
@@ -103,15 +102,48 @@ async def _set_up_justnimbus(hass: HomeAssistant, flow_id: str) -> None:
     ) as mock_setup_entry:
         result2 = await hass.config_entries.flow.async_configure(
             flow_id=flow_id,
-            user_input={
-                CONF_CLIENT_ID: "test_id",
-            },
+            user_input=FIXTURE_USER_INPUT,
         )
         await hass.async_block_till_done()
 
     assert result2["type"] == FlowResultType.CREATE_ENTRY
     assert result2["title"] == "JustNimbus"
-    assert result2["data"] == {
-        CONF_CLIENT_ID: "test_id",
-    }
+    assert result2["data"] == FIXTURE_USER_INPUT
     assert len(mock_setup_entry.mock_calls) == 1
+
+
+async def test_reauth_flow(hass: HomeAssistant) -> None:
```

Comment:
@MartinHjelmare My Python is a bit rusty :/. I know the test is incorrect but I cannot find out how to correctly test the code. Can you help me by giving me a few pointers?

Reply:
The test looks ok in general. I added some suggestions for completeness.